rpc/api/miner.go

```diff
@@ -126,7 +126,7 @@ func (self *minerApi) SetExtra(req *shared.Request) (interface{}, error) {
 		return nil, err
 	}
 
-	if uint64(len(args.Data)) > params.MaximumExtraDataSize.Uint64()*2 {
+	if uint64(len(args.Data)) > params.MaximumExtraDataSize.Uint64() {
```

Comment:
Oh ok, that explains the ```*2``` (I was quite confused as to where it came from). Sounds like an easy enough fix, but can I ask: Why should we require that the input is in hex? Based off of the official documentation [here](https://github.com/ethereum/go-ethereum/wiki/JavaScript-Console#minersetextra) and the forum posts [here](https://forum.ethereum.org/discussion/2773/help-needed-in-mining), [here](https://www.reddit.com/r/ethereum/comments/3fgiac/can_a_miner_put_a_message_into_a_block_for_me/), and [here](https://forum.ethereum.org/discussion/2684/mine-directly-into-btc) the normal way of providing input to it doesn't seem to be in hex. I'm just hesitant to break the current user experience unnecessarily. 

Would it make more sense to maintain a ```miner.setExtraHex()``` in addition to what we already have?